gnosis/conditional-markets-interface
# Feature #49, #53: Resolution Values in Sidebar

# Description
Adds resolution value to sidebar for both categorical and scalar markets. It shows various states depending on resolution value and market status.

**Market Status "Closed", Winning Outcome not set:**
> "This market has been closed but the oracle has not yet resolved the winning outcome. Please wait"

**Market Status "Closed", Winning Outcome set:**
For Categorical:
![image](https://user-images.githubusercontent.com/969493/72731768-a3f28500-3b94-11ea-8d3d-a90772e72e21.png)

For Scalar:
![image](https://user-images.githubusercontent.com/969493/72731794-a9e86600-3b94-11ea-930b-e1f625975f47.png)

# To Test
- Display a resolved, scalar market to see the resolution value and the last trade in the sidebar. No other options should be available, except redeeming, if user invested in market.
- Display a resolved, categorical market to see the resolution value in the sidebar. No other options should be available, except redeeming, if user invested in market.

# Code
I moved trade-history data out into `Root` Component, to allow passing it to both `Sidebar` and `MarketTable`. `MarketTable` displays the graph and the `Sidebar` the value and last trade data (for scalar)
I also fixed a warning that occurred because the invisible graph was rendered, which made `recharts` complain about an invalid height. Fixed this by only rendering the graph when the tab was open.
